pyre/extra_tests/snippets/call_star_unpack_collecting_iterable.py

```
@@ -0,0 +1,86 @@
+
+
+def gen(n):
+    for i in range(n):
+        gc.collect()
+        yield [i]
+
+
+class CollectingIterator:
+    """Not a generator, so the ``next()`` drain runs instead of ``unpack_into``."""
+
+    def __init__(self, n):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)